JobConfig/digitize/Extracted.fcl

```diff
@@ -24,9 +24,5 @@ physics.producers.CrvPhotons.lookupTableFileNames : ["CRVConditions/v6_0/LookupT
 # for 68 PE/SiPM @ 1 m away from SiPM (Test beam June 2017)
 # using the pulse height calibration, this value gives 45 PE/SiPM @ 1 m away from SiPM
 physics.producers.CrvPhotons.scintillationYields  : [39400,39400,39400]
```

Comment:
I'm not sure, if the CrvPhotons settings are needed here. They are already in Offline/CRVResponse/fcl/epilog_extracted.fcl - except the scintillationYields, which should probably also go to Offline/CRVResponse/fcl/epilog_extracted.fcl.